src/components/2019/year-in-review/News.js

```diff
@@ -0,0 +1,534 @@
+                                    <Date> July 2 </Date>
+                            </div>
+                            <div class="col-md-6">
+                                <img class="img-fluid"
+                                src="https://snworksceo.imgix.net/dpn/a9f9468c-3601-4dbc-b59b-13f9abd1d03e.sized-1000x1000.jpg?w=1000" />
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <MoreStories>
+                    <SubSectionHeader color={"white"}>MORE STORIES</SubSectionHeader>
+                    <div class="row">
+                        <div class="col-md-3 right-border">
+                            <StyledStoryLink href="https://www.thedp.com/article/2019/03/penn-athletics-jerome-allen-philip-esformes-trial-update-bribery-basketball-morris">
+                                <SecondaryStory> Updated: Former Penn coach Jerome Allen testifies, implicating former assistant </SecondaryStory>
+                            </StyledStoryLink>
+                            <Date> Mar. 10 </Date>
+                        </div>
+                        <div class="col-md-3 right-border narrow-col-left">
+                            <StyledStoryLink href="https://www.thedp.com/article/2019/03/ivy-league-college-admissions-bribery-justice-department-jerome-allen"  >
+                                <SecondaryStory> Penn not among schools named in nationwide college admissions scandal </SecondaryStory>
+                            </StyledStoryLink>
+                            <Date> Mar. 12 </Date>
+                        </div>
+                        <div class="col-md-3 right-border narrow-col-left">
+                            <StyledStoryLink href="https://www.thedp.com/article/2019/03/bribery-esformes-scandal-william-singer-college-admissions"  >
+                                <SecondaryStory> Parent who bribed former Penn coach contacted consultant at center of admissions scandal </SecondaryStory>
+                            </StyledStoryLink>
+                            <Date> Mar. 16 </Date>
+                        </div>
+                        <div class="col-md-3 narrow-col-left">
+                            <StyledStoryLink href="https://www.thedp.com/article/2019/07/jerome-allen-probation-accepting-bribes-penn-mens-basketball-college-admissions-scandal-esformes"  >
+                                <SecondaryStory> Former Penn coach Jerome Allen avoids jail time in admissions bribery scandal </SecondaryStory>
+                            </StyledStoryLink>
+                            <Date> July 2 </Date>
+                        </div>
+                    </div>
+                </MoreStories>
+            </div>
+        </Section>
+
+        <Section inverse={true}>
```

Comment:
instead of inverse={true} you can just put inverse, it resolves to inverse={true} automatically